src/Microsoft.Framework.TestHost/Program.cs

```diff
@@ -22,7 +23,9 @@ public Program(IServiceProvider services)
 
         public int Main(string[] args)
```

Comment:
Right now there's a bunch of vestigial stuff in this class to support older VS builds. Once VS goes RTM, I'll remove the old support.